tools/basecurve/CMakeLists.txt

```diff
@@ -1,4 +1,10 @@
-find_package(OpenMP)
+message("darktable-curve-tool: USE_OPENMP: ${USE_OPENMP}")
+if (USE_OPENMP)
+    find_package(OpenMP)
+else()
+    set(OPENMP_FOUND FALSE)
```

Comment:
I wouldn't bother with this and the extra messages, "find_package" is verbose enough...

Reply:
> I wouldn't bother with this and both of the extra messages, "find_package" is verbose enough...
> 
> The gate by USE_OPENMP is ok.

Sounds good! Update pushed.